src/datachain/cache.py

```diff
@@ -86,9 +84,11 @@ async def download(
         size = file.size
         if size < 0:
             size = await client.get_size(from_path, version_id=file.version)
+        from tqdm.auto import tqdm
+
         cb = callback or TqdmCallback(
             tqdm_kwargs={"desc": odb_fs.name(from_path), "bytes": True, "leave": False},
-            tqdm_cls=Tqdm,
```

Comment:
@skshetry do we really need that whole custom class to support this?

Reply:
This was copied from `dvc-objects` which in turn came from `dvc`. There are some minor improvements (see how it sets bar_format, and fills the `ncols`, etc.), but it is debatable if it's needed. 

I believe we should have a uniform Tqdm progress bars, so I do think we should have a small class or a function that gives us similar looking progress bars.

Reply:
then I suggest we come to it later when we have actual demand / improvements

it's strange to have a large class that is used in a single internal place